configs/recognition/omnisource/README.md

```diff
@@ -17,7 +17,100 @@ We currently released 4 models trained with OmniSource framework, including both
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-c3ow{border-color:inherit;text-align:center;vertical-align:top}
+</style>
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-c3ow">Model</th>
+    <th class="tg-c3ow" colspan="3">Baseline</th>
+    <th class="tg-c3ow" colspan="3">+GG-img</th>
+    <th class="tg-c3ow" colspan="3">+[GG-IG]-img</th>
+    <th class="tg-c3ow" colspan="3">+IG-vid</th>
+    <th class="tg-c3ow" colspan="3">+KRaw</th>
+    <th class="tg-c3ow" colspan="3">OmniSource</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-c3ow" rowspan="2">TSN-8seg<br>ResNet50</td>
+    <td class="tg-c3ow" colspan="3">77.4 / 93.6</td>
+    <td class="tg-c3ow" colspan="3">78.0 / 93.6</td>
+    <td class="tg-c3ow" colspan="3">78.6 / 93.6</td>
+    <td class="tg-c3ow" colspan="3">80.6 / 95.0</td>
+    <td class="tg-c3ow" colspan="3">78.6 / 93.2</td>
+    <td class="tg-c3ow" colspan="3">81.3 / 94.8</td>
+  </tr>
+  <tr>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+  </tr>
+  <tr>
+    <td class="tg-c3ow" rowspan="2">SlowOnly-8x8<br>ResNet50</td>
+    <td class="tg-c3ow" colspan="3">78.6 / 93.9</td>
+    <td class="tg-c3ow" colspan="3">80.8 / 95.0</td>
+    <td class="tg-c3ow" colspan="3">81.3 / 95.2</td>
+    <td class="tg-c3ow" colspan="3">82.4 / 95.6</td>
+    <td class="tg-c3ow" colspan="3">80.3 / 94.5</td>
+    <td class="tg-c3ow" colspan="3">82.9 / 95.8</td>
+  </tr>
+  <tr>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+  </tr>
+</tbody>
+</table>
```

Comment:
Sth wrong with the CSS, https://github.com/open-mmlab/mmaction2/blob/7d5bba27609c8d86d90d7765de6d9b9a303c7fa1/configs/recognition/omnisource/README.md

Reply:
OK, I will remove this part.